docs/data-modeling/admin-data-model.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Data modeling for admins"
+redirect_from:
+  - /docs/latest/administration-guide/03-metadata-editing
+---
+
+# Data modeling for admins
+
+Let's say you're counting active users. Do you use "count of users", "user count", or "distinct users"? Should we present those counts as "10,000", "10K", or "10000"? And what does "active" really mean, anyway?
+
+When you're [pushing and pulling data](https://www.metabase.com/learn/analytics/push-and-pull) between different teams, you're bound to run into different names, formats, and definitions for the same thing. Enter the **Admin Data Model**: a tiny command centre for all of the business logic living in your Metabase. Don't let the power get to your head.
+
+## Quick start
```

Comment:
This heading shouldn't exist, and the list should kick off the doc (see my notes in the main comment).